src/components/Layout/Layout.tsx

```diff
@@ -26,32 +26,49 @@ import { LEFT_NAV_LINKS, RIGHT_NAV_LINKS } from '@/utils/globalnav';
 import { trackPageVisit } from '../../utils/track';
 import { Menu } from '@/components/Menu';
 import { LayoutProvider } from '@/components/Layout';
+import { TableOfContents } from '@/components/TableOfContents';
+import type { Heading } from '@/components/TableOfContents/TableOfContents';
 import directory from 'src/directory/directory.json';
 import { PageNode } from 'src/directory/directory';
 
-export const Layout = forwardRef(function Layout(
-  {
-    children,
-    pageTitle,
-    pageDescription,
-    platform,
-    url,
-    pageType = 'inner'
-  }: {
-    children: any;
-    pageTitle?: string;
-    pageDescription?: string;
-    platform?: Platform;
-    url?: string;
-    pageType?: 'home' | 'inner';
-  },
-  footerRef
-) {
+export const Layout = ({
+  children,
+  hasTOC = true,
```

Comment:
I have some logic to not output the ToC if there are no headings found, but I thought we might run into some cases where we still want to hide the ToC so I added a prop here. Pages that we don't want to show a ToC (regardless of headings found) we can pass a static prop to turn it off.